[2025-03-23T00:34:14.113+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-23T00:34:14.124+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_pipeline.load_to_hdfs scheduled__2025-03-22T00:00:00+00:00 [queued]>
[2025-03-23T00:34:14.130+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_pipeline.load_to_hdfs scheduled__2025-03-22T00:00:00+00:00 [queued]>
[2025-03-23T00:34:14.130+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 4
[2025-03-23T00:34:14.142+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load_to_hdfs> on 2025-03-22 00:00:00+00:00
[2025-03-23T00:34:14.145+0000] {standard_task_runner.py:72} INFO - Started process 616910 to run task
[2025-03-23T00:34:14.147+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'fraud_detection_pipeline', 'load_to_hdfs', 'scheduled__2025-03-22T00:00:00+00:00', '--job-id', '144', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmpk92gnypt']
[2025-03-23T00:34:14.149+0000] {standard_task_runner.py:105} INFO - Job 144: Subtask load_to_hdfs
[2025-03-23T00:34:14.174+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_pipeline.load_to_hdfs scheduled__2025-03-22T00:00:00+00:00 [running]> on host usmail-ThinkBook-14-G2-ITL
[2025-03-23T00:34:14.218+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ismailsamilacc@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fraud_detection_pipeline' AIRFLOW_CTX_TASK_ID='load_to_hdfs' AIRFLOW_CTX_EXECUTION_DATE='2025-03-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-22T00:00:00+00:00'
[2025-03-23T00:34:14.219+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-23T00:34:16.658+0000] {logging_mixin.py:190} WARNING - creating folder in HDFS
[2025-03-23T00:34:16.658+0000] {dags.py:90} INFO - creating folder in HDFS
[2025-03-23T00:34:16.662+0000] {logging_mixin.py:190} WARNING - Copying data to data into /hadoop_namenode mounted volume
[2025-03-23T00:34:16.662+0000] {dags.py:93} INFO - Copying data to data into /hadoop_namenode mounted volume
[2025-03-23T00:34:18.399+0000] {logging_mixin.py:190} WARNING - Data loaded to HDFS
[2025-03-23T00:34:18.398+0000] {dags.py:96} INFO - Data loaded to HDFS
[2025-03-23T00:34:18.400+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-23T00:34:18.407+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-23T00:34:18.408+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_pipeline, task_id=load_to_hdfs, run_id=scheduled__2025-03-22T00:00:00+00:00, execution_date=20250322T000000, start_date=20250323T003414, end_date=20250323T003418
[2025-03-23T00:34:18.454+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-23T00:34:18.484+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-23T00:34:18.489+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
